src/features/auth/pages/signup/SetEmail/SetEmail.native.test.tsx

```diff
@@ -142,4 +143,20 @@ describe('<SetEmail />', () => {
 
     expect(analytics.logHasCorrectedEmail).toHaveBeenNthCalledWith(1, { from: 'setemail' })
   })
+
+  it('should navigate to Login with offerId when in navigation params clicking on "Se connecter" button', async () => {
```

Comment:
Je comprends pas bien ce titre de test, je pense que tu peux le simplifier
"should navigate to Login with provided offerId when clicking on "Se connecter"" si c'est bien ça que tu veux faire ?

Reply:
Effectivement je me suis pas relu, merci !